File: angular-food-delivery-website/src/app/user-registration-dto.ts
export class UserRegistrationDto {
  firstName: string
  lastName: string
  gender: Gender
  email: string
  password: string
  passwordConfirmation: string
  phoneNumber: string
}

export enum Gender {
    Male = 1,
    Female = 2
  }
  
  export const GenderEnumLocalization = {
    [Gender.Male]: 'Мъж',
    [Gender.Female]: 'Жена'
  };

File: FoodDeliveryWebsite/FoodDeliveryWebsite.Models/Dtos/UserDtos/UserRegistrationDto.cs
using FoodDeliveryWebsite.Models.Entities;
using FoodDeliveryWebsite.Models.Enums;
namespace FoodDeliveryWebsite.Models.Dtos.UserDtos
{
    public class UserRegistrationDto
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public Gender Gender { get; set; }

        public string Email { get; set; }

        public string Password { get; set; }

        public string PasswordConfirmation { get; set; }

        public string PhoneNumber { get; set; }

        public List<Address> Addresses { get; set; }
    }
}
